### ISSUE TITLE
Microwave frequencies

### ISSUE THREAD

**Issue by alberto-tf** [opticspy/lightpipes#73]
Could LightPipes be used at microvave frequencies?. I am thinking of using Lightpipes for simulating with Gaussian Beams the propagation through several mirrors in a Beam-waveguide antenna.

**Comment by FredvanGoor** [comment]
Yes and no. Far enough from the antenna I expect that LightPipes can be used. Near the antenna the magnetic and electric fields are more complex and cannot be described with LightPipes. Look at antenna theory and things like Fraunhofer distances.

**Comment by FredvanGoor** [comment]
Gaussian beams in a waveguide antenna? What is that?

**Comment by alberto-tf** [comment]
Take a look at the following article:

https://ntrs.nasa.gov/archive/nasa/casi.ntrs.nasa.gov/19940025121.pdf
